pegasus/sites.v3/code.org/views/social_media.haml

```diff
@@ -1,6 +1,7 @@
 - facebook ||= nil
 - twitter ||= nil
 - instagram ||= nil
+- tiktok ||= nil
```

Comment:
Not blocking, but you can use css here instead of `&nbsp;` to create the spacing between the icons:
```suggestion
- tiktok ||= nil

:css
  .flex-container {
    display: flex;
    gap: 1rem;
  }

.flex-container
  - unless facebook.nil?
    %a{href: "https://www.facebook.com/#{facebook}", target: "_blank"}
      %i.fa.fa-facebook.fa-lg
  - unless twitter.nil?
    %a{href: "https://twitter.com/#{twitter}", target: "_blank"}
      %i.fa.fa-twitter.fa-lg
  - unless instagram.nil?
    %a{href: "https://www.instagram.com/#{instagram}", target: "_blank"}
      %i.fa.fa-instagram.fa-lg
  - unless tiktok.nil?
    %a{href: "https://www.tiktok.com/@#{tiktok}", target: "_blank"}
      %i.fa-brands.fa-tiktok.fa-lg

```

Reply:
<img width="829" alt="Screenshot 2024-02-23 at 2 14 52 PM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/75089739-b52b-4ba9-bcfb-0227c2a1aadf">

Reply:
Looks great, thanks for updating!